src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -281,9 +281,71 @@ def dcos_install_cli(install_location=None, client_version='1.8'):
         raise CLIError('Connection error while attempting to download client ({})'.format(err))
 
 
+def k8s_install_draft(client_version='v0.5.1', install_location=None):
+    base_url = 'https://github.com/Azure/draft/releases/download/{}/draft-{}-{}-amd64.tar.gz'
```

Comment:
We already have releases pushed to WABS.

https://azuredraft.blob.core.windows.net/draft/draft-v0.6.0-linux-amd64.tar.gz